driver/src/main/java/org/neo4j/driver/TransactionConfig.java

```diff
@@ -202,7 +202,7 @@ public Builder withTimeout( Duration timeout )
         public Builder withMetadata( Map<String,Object> metadata )
         {
             requireNonNull( metadata, "Transaction metadata should not be null" );
-
+            String hello = "hello";
```

Comment:
just noticed I pushed this line mistake (thought git stash cleared everything)

Reply:
fixed

Reply:
Declining the pr to re-open to the official fork